.github/actions/setup/action.yaml

```
@@ -8,7 +8,7 @@ runs:
         sudo apt-get install -y git-restore-mtime
       shell: bash
     - name: setup blogsync
-      uses: x-motemen/blogsync@v0
+      uses: x-motemen/blogsync@v0.13.5
       with:
         version: latest
```

setup action自体のバージョンではなく、setup actionにわたす引数のバージョンを変更する必要がありそうです
```suggestion
      uses: x-motemen/blogsync@v0.13.5
      with:
        version: "v0.13.5"
```